CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
-# Contributing to Glutil
+# Contributing to `glutil`
```

Comment:
Forking the project repo and creating a PR from the forked repo's feature branch into the original repo seems like a more common contribution mechanism for open-source projects in general, but I do like the simplicity of this approach.

Reply:
I think the fork/PR flow is implicit because of how github operates. I looked at some other repos contributing docs and most of them (including large repos with lots of external contributors) do not cover that.

as for changing Glutil to `glutil`, I don't think it should be changed, as elsewhere the project is referred to as Glutil without formatting (specifically the title of the readme).

Reply:
Is it implicit? I cloned the original repo, did not fork it, and submitted a PR to the original repo.  In my experience, when a fork is required, it's because Git permissions are restricted on the original repo.

Reply:
Yes it's permissions, but the "github flow" that they advertise and that most repo that aren't worked on by a single entity is the fork and pull method because it's the only way to contribute to repositories you don't own or that are owned by an organization you're not part of.

See [Github's intro article on forking](https://guides.github.com/activities/forking/) and [Github's Contributing to Open Source guide](https://opensource.guide/how-to-contribute/#opening-a-pull-request)